resources/js/Pages/Dashboard/Timesheet/Index.vue

```diff
@@ -387,12 +363,15 @@ export default {
     },
 
     addBlankTimesheetRow() {
+      let project = this.projects[this.projects.findIndex(p => p.value === this.form.project)];
```

Comment:
@djaiss you can see `this.form.project` contains now only the 'value' (code) of the project